thirdparty/fonts/LICENSE_Mononoki

```diff
@@ -1,8 +1,9 @@
-Copyright 2010, 2012 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name 'Source'. All Rights Reserved. Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries.
```

Comment:
Please update `thirdparty/README.md` and `COPYRIGHT.txt` too.

Reply:
could not find anything in README.md

Reply:
https://github.com/godotengine/godot/blob/master/thirdparty/README.md#fonts

Reply:
damn, should be fixed now.